documentation/src/main/asciidoc/ch09.asciidoc

```diff
@@ -370,6 +370,86 @@ include::{sourcedir}/org/hibernate/validator/referenceguide/chapter09/Bootstrapp
 Refer to <<section-fail-fast>> and <<section-programmatic-api>> to learn more about the fail fast
 mode and the constraint declaration API.
 
+[[section-script-evaluator-factory]]
+==== Configuring script evaluator factory (`ScriptEvaluatorFactory`)
+
+For constraints like `@ScriptAssert` and `@ParameterScriptAssert`, it might be useful to configure
+how script evaluators (`ScriptEvaluator`) are build, and which scripting engines are used to build
+them. This can be done by setting a custom implementation of `ScriptEvaluatorFactory`. In
+particular this is important for OSGi environments (or similar to it) where user might face issues
```

Comment:
"this is important for modular environments (e.g. OSGi)" instead of "this is important for OSGi environments (or similar to it)"